src/platforms/javascript/common/performance/index.mdx

```diff
@@ -119,7 +119,7 @@ Once the data is included in the `<meta>` tag, our `BrowserTracing` integration
 
 ### Navigation and Other XHR Requests
 
-Once a page is loaded, any requests it makes and any responses your backend generates as a result are linked through a request/response header.
+Once a page is loaded, any requests it makes (and any requests your backend makes as a result) are linked through a request header.
```

Comment:
This statement can be misleading. Trace propagation via request headers only happens when there is an "active" transaction / span.

Since we're at it, I'd suggest:

```suggestion
Once a page is loaded, every navigation event triggers a `navigation` transaction, and any Fetch or XHR requests made within this transaction will automatically propagate the trace information.
```

Feel free to tweak it.